generators/app/templates/infrastructure/src/components/menu/MenuItem.js

```diff
@@ -61,7 +61,8 @@ MenuItem.propTypes = {
   activeRoute: PropTypes.func,
   isSubMenuItem: PropTypes.bool,
   subMenuOpen: PropTypes.bool,
-  onToggleSubMenu: PropTypes.func
+  onToggleSubMenu: PropTypes.func,
+  withGradient: PropTypes.bool
```

Comment:
In some places you defined this new property as required, here is not...which is it? Is required or not? Please keep the consistency.